nowcasting_dataset/config/load.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import gcsfs
+import os
+import io
+import yaml
+from nowcasting_dataset.config.model import Configuration
+
+logger = logging.getLogger(__name__)
+
+
+def load_yaml_configuration(filename: str) -> Configuration:
+    """
+    Load a yaml file which has the a configruation in it
```

Comment:
Tiny typo fix :)   Sorry for being such a pedant!

```suggestion
    Load a yaml file which has a configuration in it
```

Reply:
no good to push me on that, my typos / spell is not very good